python/lsst/ts/xml/data/sal_interfaces/ATBuilding/ATBuilding_Events.xml

```diff
@@ -50,4 +51,52 @@
       <Count>1</Count>
     </item>
   </SALEvent>
+  <SALEvent>
+    <Subsystem>ATBuilding</Subsystem>
+    <EFDB_Topic>ATBuilding_logevent_ventGateLimitSwitchFullyClosed</EFDB_Topic>
```

Comment:
Instead of doing an event with a boolean for closed/open, consider an enumeration with states "OPENED", "CLOSED", "PARTIALLY_OPEN".

Reply:
Sure.  For clarification, then: it is preferred to represent complex/composite states, rather than primitives like single limit switch states, at this level?